tasks/task_14/client/sender/sender.go

```diff
@@ -0,0 +1,36 @@
+package sender
+
+import (
+	"fmt"
+	"net"
+)
+
+type Sender struct {
+	conn    net.Conn
+	scanner scanner
+}
+
+func NewSender(conn net.Conn, s scanner) *Sender {
+	return &Sender{
+		conn:    conn,
+		scanner: s,
+	}
+}
+
+type scanner interface {
+	ScanTerminal() (string, error)
+}
+
+func (s *Sender) SendStringToCheck() error {
+	str, err := s.scanner.ScanTerminal()
+	if err != nil {
+		return fmt.Errorf("error while scanning terminal")
+	}
+
+	_, err = s.conn.Write([]byte(str + "\n"))
```

Comment:
shouldn't this "\n" be added by the scanner? what if scanned will add it also? what if "\n" will be added two times?